### ISSUE TITLE
Deploy on real infrastructure

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#60]
Docker container exists: folks in Pittsburgh should deploy this along with all the other containers, using whatever configuration management they are using.

**Comment by ngehlenborg** [comment]
@yuanzhou @shirey @mccalluc: is there already some agreement about how we will be able to deploy UI updates when we have updated the UI docker container?

**Comment by mccalluc** [comment]
No, not formally -- Some possibilities, in order from most to least preferred:
- There is a repo for a HuBMAP docker compose: I make a PR to bump the version number of the portail-ui image as needed.
- I file an issue somewhere, asking for a new  image to be deployed.
- I slack someone, asking for the the new image to be deployed

**Comment by ngehlenborg** [comment]
Thanks @mccalluc. I will add any further comments to https://github.com/hubmapconsortium/hubmap-data-portal/issues/51.

**Comment by mccalluc** [comment]
@yuanzhou : Let me know if there's anything I can do to help move this along.

**Comment by yuanzhou** [comment]
Just an update, @mccalluc and I worked on deploying the portal-ui on our testing AWS server this afternoon. I've created SSL certificates for the portal-i domain and it works well with other containerized services. We ran into a globus redirect issue that requires some tweaks. And we are also working on a workflow to facilitate the future code updates -> deployment process. Any suggestions or concerns are welcome.
